test/REQUIRE

```diff
@@ -1,4 +1,3 @@
 Combinatorics
 Clp
-CDDLib
```

Comment:
I need to change something on CDDLib with the new version of Polyhedra. I will reenable it once Polyhedra is tagged and then CDDLib is tagged